core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/CustomImageCatalogV4Controller.java

```diff
@@ -48,11 +51,13 @@
     @Inject
     private ConverterUtil converterUtil;
 
+    @Inject
+    private ImageCatalogFiltering imageCatalogFiltering;
+
     @Override
-    @FilterListBasedOnPermissions(action = AuthorizationResourceAction.DESCRIBE_IMAGE_CATALOG, filter = ImageCatalogFiltering.class)
+    @FilterListBasedOnPermissions
```

Comment:
Ohh, I didn't know about this class.

Reply:
it is a new controller, created by sloth team, but filtering was missing here, thus i added an enforce logic for this annotation